pdks/FinFET14nm_Mock_PDK/layers.json

```diff
@@ -461,7 +461,7 @@
 "VencA_H": 60,
 "VencP_L": 0,
 "VencP_H": 472,
-"ViaCut":{"Gen":"Array", "WidthX":256, "WidthY":256, "SpaceX":360, "SpaceY":360, "Number":[2,3]},
+"ViaCut":{"Gen":"ViaArrayGenerator", "WidthX":256, "WidthY":256, "SpaceX":360, "SpaceY":360, "Number":[3,2]},
```

Comment:
@arvuce22 I want to make sure you meant [3, 2] and not[2, 3] here. I also feel like we are mixing terminologies. WidthX, WidthY but using a list for number. Its all configurable so its not a big issue but we should settle on an API that makes sense.

Reply:
![image](https://user-images.githubusercontent.com/48996821/79938947-c655a200-8412-11ea-9805-12b1e8949d6e.png)
is the generated intersection with [3,2]

Reply:
I defined "Number" as [number of rows, number of columns], therefore, for the above figure, it's [2,3].
We can change this other way:  [number of columns, number of rows]

Reply:
We can also be explicit and say NumX, NumY.

Reply:
Sounds good! NumX, NumY can be used

Reply:
Done